vtex/omnishipping
# Fixes selecting addresses priority

#### What is the purpose of this pull request?
Fixes selecting addresses priority

#### What problem is this solving?
Closes [story](https://app.clubhouse.io/vtex/story/24197/omnishipping-selecionando-endere%C3%A7o-do-cliente-ao-inv%C3%A9s-de-manter-o-endere%C3%A7o-da-giftlist)

#### How should this be manually tested?
1. Add [Items to cart](http://fernando--vtexgame1.vtexcommercestable.com.br/checkout/cart/add?&workspace=fernando&sku=31&qty=1&seller=1&sc=1&gr=21)
2. Identify as `second-purchase@mailinator.com`
3. Finish purchase and address should be the `Teste Endereço`

#### Screenshots or example usage
n/a

#### Types of changes
